### DISCUSSION TITLE
Detect that requested content has been watched--automate deletion

### DISCUSSION THREAD

**Discussion by Minituff** [sct/overseerr#2480] [Feature Requests]
I have users who request content that no one else will watch. I am happy to download these movies/shows, but I don't want to keep them on my Plex server after they have completed it. I would love a feature where the user could submit to me that they have completed the show/movie and I can delete it.

Ideas:
- [ ] Leverage the upcoming the Tautulli API to automatically flag shows that have been watched by their `requester`
- [ ] Allow the requester to submit `Watched` status for each movie/show so that I can remove it from my server. This feature could be piggy-packed off the new `Issues` feature.
- [ ] Bonus points if automation is added to delete this content automatically if certain criteria is met. For example: If the requester has not watched for 6 months, then delete. Or if the requester has finished the last episode of the show, then delete (after x time)

**Comment by JaiMistry** [comment]
In the same boat here, would love a feature like this to help automate freeing up some storage

**Comment by TheCatLady** [comment]
Deletion of media will not be added to Overseerr.  See #1924.

**Reply by Minituff** [reply]
@TheCatLady I understand the decision to not implement media deletion.

I think bullet points 1 and 2 would still be a great feature. If a user wants to tell me they've already watched something they requested, they have to open an issue in the "other" category just to tell me it's been watched. Building auto-detection using Tautulli would probably be the most elegant solution since it looks like this feature already does most of the heavy lifting.

As for point 3, maybe we could have a "Pending deletion" section that lists media that is marked for manual upcoming deletion. This could be something I could check on to get see content I no longer need on my server. After I delete the media on the server, Overseerr would refresh and remove those items from the list. We could still have some of the automation I mentioned early before its added to the "Pending deletion" list.

**Reply by TheCatLady** [reply]
Detecting that media is no longer available on the media server is already in the works.

As for the watch status request, there is the issue that there is only watch data for Plex users, so the lack of watch data isn't really a valid data point.  Also, it'd be expensive to check the watch history of each request as the list of requests / available media grows.

The existing feature does not support what you ask for; it simply fetches the watch data as needed based on client-side actions.  A window into Tautulli data so you don't always need to open Tautulli, really.
